homeassistant/const.py

```diff
@@ -184,6 +184,7 @@
 EVENT_CORE_CONFIG_UPDATE = "core_config_updated"
 EVENT_HOMEASSISTANT_CLOSE = "homeassistant_close"
```

Comment:
Maybe we should rename this event in that case to `EVENT_HOMEASSISTANT_STOPPED`

Reply:
Too late :) I think close is fine.